purchasing_test/integration/opportunities/test_opportunities_front.py

```diff
@@ -250,3 +252,16 @@ def test_manage_subscriptions(self):
 
         self.assert200(unsubscribe_all)
         self.assertTrue('You are not subscribed to anything!' in unsubscribe_all.data)
+
+class TestExpiredOpportunities(TestOpportunitiesBase):
+    render_templates = True
+
+    def setUp(self):
+        super(TestOpportunitiesBase, self).setUp()
+
+    def test_expired_opportunity(self):
+        '''Tests that expired view works
+        '''
+
+        expired = self.client.get('/beacon/opportunities/expired')
+        self.assert200(expired)
```

Comment:
You'll want to test more here -- use the context variable check to test that expired opportunities really show up. One way to do this would be to take one of the four opportunities you have available in `TestOpportunityBase` and make its `planned_submission_end` field be yesterday (`datetime.datetime.today() - datetime.timedelta(days=1)`.